test/ArbitrumFoundationVestingWallet.t.sol

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "../src/ArbitrumFoundationVestingWallet.sol";
+import "../src/L2ArbitrumToken.sol";
+import "../src/L2ArbitrumGovernor.sol";
+import "../src/ArbitrumTimelock.sol";
+import "../src/Util.sol";
+
+import "./util/TestUtil.sol";
+
+import "forge-std/Test.sol";
+
+contract ArbitrumFoundationVestingWalletTest is Test {
```

Comment:
would be good to add a test case that forks testnet/mainnet

Reply:
we have a separate repo w/ a governance simulation seatbelt, we can add it there

Reply:
isn't the point of the seatbelt to simulate proposals that are up? i think its works as a good test case here the path for changing the vesting schedule if needed

Reply:
i think seatbelt only shows the sideeffects that happen from running a proposal (ie events emitted, storage changed, etc) - it doesnt actually test if any of the functionality works after running this